closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -887,101 +997,102 @@ boolean isRestrictedProviderAllowed(String providerName) {
         /**
          * List audit info if userSecurityAudit is true, default as false.
          */
-        private void listAudit() {
+        private void listAvailableProfiles() {
             System.out.println();
-            System.out.println("Restricted Security Audit Info:");
-            System.out.println("===============================");
-
-            for (int num = 1;; ++num) {
-                String desc = securityProps.getProperty("RestrictedSecurity" + num + ".desc.name");
-                if ((desc == null) || desc.trim().isEmpty()) {
-                    break;
+            System.out.println("Restricted Security Available Profiles' Info:");
+            System.out.println("=============================================");
+
+            Set<String> availableProfiles = new HashSet<>();
+            for(Object securityFileObject : securityProps.keySet()) {
+                if (securityFileObject instanceof String key) {
+                    Pattern profileNamePattern = Pattern.compile("^(RestrictedSecurity\\.\\S+)\\.desc\\.name");
+                    Matcher profileMatcher = profileNamePattern.matcher(key);
+                    if(profileMatcher.matches()) {
+                        availableProfiles.add(profileMatcher.group(1));
+                    }
                 }
-                System.out.println("RestrictedSecurity" + num + ".desc.name: "
-                        + securityProps.getProperty("RestrictedSecurity" + num + ".desc.name"));
-                System.out.println("RestrictedSecurity" + num + ".desc.number: "
-                        + parseProperty(securityProps.getProperty("RestrictedSecurity" + num + ".desc.number")));
-                System.out.println("RestrictedSecurity" + num + ".desc.policy: "
-                        + parseProperty(securityProps.getProperty("RestrictedSecurity" + num + ".desc.policy")));
-                System.out.println("RestrictedSecurity" + num + ".desc.sunsetDate: "
-                        + parseProperty(securityProps.getProperty("RestrictedSecurity" + num + ".desc.sunsetDate")));
-                System.out.println();
+            }
+            System.out.println("Those are the available Restricted Security profiles:\n");
```

Comment:
Those -> These

Reply:
This could be "The available Restricted Security profiles:"

Reply:
Changed to your suggestion.